Pinta.Core/Managers/ToolManager.cs

```diff
@@ -147,7 +155,7 @@ void HandlePbToolItemClicked (object? sender, EventArgs e)
 		public void Commit ()
 		{
 			if (CurrentTool != null)
-				CurrentTool.DoCommit ();
+				CurrentTool.DoCommit (PintaCore.Workspace.HasOpenDocuments ? PintaCore.Workspace.ActiveDocument : null);
```

Comment:
Just a thought - would it make sense to change `ActiveDocument` to return a `Document?` and avoid lots of conditional checks of `HasOpenDocuments`?

Reply:
It's probably a little better, however it just moves the `HasOpenDocuments` check to an `is null` check.  But now that the compiler will force you to check for `null` it would be fine.  Let's avoid it in this PR, as there's ~150 calls to `ActiveDocument`.  😉

Reply:
Agreed, I think that can be left out of this PR.